data/sql/updates/pending_db_world/rev_1677501542428607570.sql

```diff
@@ -0,0 +1,743 @@
+SET @SORCERER_ID = 4294;
+SET @MYRMIDON_ID = 4295;
+SET @DEFENDER_ID = 4298;
+SET @CHAPLAIN_ID = 4299;
+SET @WIZARD_ID = 4300;
+SET @CENTURION_ID = 4301;
+SET @CHAMPION_ID = 4302;
+SET @ABBOT_ID = 4303;
+SET @MONK_ID = 4540;
+
+SET @RP_TEXTS_GROUP_ID := 0;
+
+SET @SAVE_THIS_WORLD_ID := 0;
+SET @SAVE_THIS_WORLD_BROADCAST_ID := 12381;
+SET @SAVE_THIS_WORLD_TEXT := (SELECT `FemaleText` FROM `broadcast_text` WHERE `ID`=@SAVE_THIS_WORLD_BROADCAST_ID);
```

Comment:
Please dont use SELECT queries unless necessary. Here you're just returning one string, so the select query is useless. Update them all to the value instead.